sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -404,6 +406,14 @@ export class AssessmentController {
           saveAssessmentCalculationStartDate,
           getProgramYearTotalAwards,
         ]);
+        // Fetch the latest CSLP balance for the student.
+        const studentLoanBalance =
+          await this.studentLoanBalanceSharedService.getLatestCSLPBalance(
+            studentId,
+          );
+        result["latestCSLPBalance"] = studentLoanBalance[0]
+          ? studentLoanBalance[0].cslBalance
+          : 0;
```

Comment:
This is the logic that the below comment can help to simplify.
https://github.com/bcgov/SIMS/pull/3121/files#r1567848943